l1-contracts/contracts/bridgehub/MessageRootBase.sol

```diff
@@ -98,37 +93,20 @@ contract MessageRoot is IMessageRoot, Initializable {
     /// @notice Checks that the message sender is the specified ZK Chain.
     /// @param _chainId The ID of the chain that is required to be the caller.
     modifier onlyChain(uint256 _chainId) {
-        if (msg.sender != BRIDGE_HUB.getZKChain(_chainId)) {
-            revert OnlyChain(msg.sender, BRIDGE_HUB.getZKChain(_chainId));
+        if (msg.sender != _bridgehub().getZKChain(_chainId)) {
+            revert OnlyChain(msg.sender, _bridgehub().getZKChain(_chainId));
         }
         _;
     }
 
-    /// @notice Checks that the Chain ID is not L1 when adding chain batch root.
-    modifier onlyL2Chain() {
```

Comment:
this was an important security feature to allow appending chain batch roots on L1 only

Reply:
it is not blocking, but better to be fixed